### DISCUSSION TITLE
Script tag for trustlogo in custom component is not executed

### DISCUSSION THREAD

**Discussion by gopher-my** [vercel/next.js#17383] [Help]
I would like to add following script in my footer (custom) component.
`<script language="JavaScript" type="text/javascript">
                            TrustLogo("https://mywebsite.com/ssl-logo/positivessl_trust_seal_sm_124x32.png", "CL1", "none");
</script>`
This script is supposed to show ssl logo image in my webpage footer component. But nothing's happened. 
I already added NextScript component in my _document.js. And I tried according to ##17358 but still not working. 

So, I created a new script file (called hello.js) which includes simple test script
`<script type="text/javascript">
console.log("*************************************************** external external");
</script>`
and try to import in my component by using
`<script type="text/javascript" src="./hello.js"></script>`
I did get any console output either. It seems no script is executed inside my custom component.
Is there anyways to show trustlogo in my custom component?
Thanks in advanced

**Comment by jamesmosier** [comment]
Hi @gopher-my. In order for this code to be run when the component is hydrated in the browser, it is best to run it like this:

```js
<script
  dangerouslySetInnerHTML={{
    __html: `
      TrustLogo("https://mywebsite.com/ssl-logo/positivessl_trust_seal_sm_124x32.png", "CL1", "none");
    `,
  }}
/>
```

The [google-analytics](https://github.com/vercel/next.js/blob/8eccecb35e8ed1bbaa0f058edc561909b83ea9e9/examples/with-google-analytics/pages/_document.js#L15-L27) example uses this method as well.
